Mage/Task/BuiltIn/Deployment/Strategy/TarGzTask.php

```diff
@@ -132,7 +156,10 @@ public function run()
     protected function excludesListFile($excludesFile)
     {
         $excludesListFileRsync = '';
-        if (!empty($excludesFile) && file_exists($excludesFile) && is_file($excludesFile) && is_readable($excludesFile)) {
+        if (!empty($excludesFile)
+            && file_exists($excludesFile)
+            && is_file($excludesFile)
+            && is_readable($excludesFile)) {
```

Comment:
Hello. This is like I found it, i have only change it to solve PSR-2 warnings. AFAIK, you're right.